CODING_STYLE.md

```diff
@@ -0,0 +1,172 @@
+# Zilliqa Coding Style
+<!--Source: https://github.com/ethereum/aleth/blob/master/CODING_STYLE.md -->
+## Code Formatting
+
+Use clang-format tool to format your changes. See [CONTRIBUTING](CONTRIBUTING.md) for details.
+
+
+## Namespaces
+
+1. No `using namespace` declarations in header files.
+
+```cpp
+       // WRONG:
+       #include <cassert>
+       using namespace std;
+       tuple<float, float> meanAndSigma(vector<float> const& _v);
+
+       // CORRECT:
+       #include <cassert>
+       std::tuple<float, float> meanAndSigma(std::vector<float> const& _v);
+```
+2. Preprocessor symbols should be prefixed with the namespace in all-caps and an underscore.
+
+## Preprocessor
+
+1. File comment is always at top, and includes:
+   - Copyright.
+   - License.
+   
+2. Use Zilliqa Standardized Guard for header files which is given by https://google.github.io/styleguide/cppguide.html#The__define_Guard.
+```
+  ZILLIQA_<path_to_header_file>_H_
+```
+3. Prefer static const variable to value macros.
+4. Prefer inline constexpr functions to function macros.
+
+
+## Capitalization
+
+GOLDEN RULE: Preprocessor: ALL_CAPS; C++: camelCase.
+
+1. Use camelCase for all class/structure names.
+2. All preprocessor symbols (macros, macro arguments) in full uppercase with underscore word separation.
+
+
+All other entities' first alpha is lower case.
+
+
+## Variable prefixes
```

Comment:
How about for global const variables (such as the variables mapped to constant.xml values)?